packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -1165,6 +1174,14 @@ class SemanticsProperties extends DiagnosticableTree {
   /// [maxValueLength] is set.
   final int? currentValueLength;
 
+  /// Provides an identifier for UI testing. This value is not exposed to the
+  /// user.
+  ///
+  /// On Android, this will become the node's `resource-id`.
+  ///
+  /// On iOS, this will become the node's `accessibilityIdentifier`.
```

Comment:
This should mention the `node` is referring to UIAccessibilityElement

Reply:
Fixed in 0ff122bfe6ec635231661e76c62004834f845493